composer.json

```diff
@@ -42,5 +48,11 @@
         "exclude-from-classmap": [
             "Tests/"
         ]
-    }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url":  "git@github.com:dayspring-tech/PropelBundle.git"
```

Comment:
@ssitu-dayspring It looks like Travis is failing when trying to clone this. I wonder if using `https://github.com/dayspring-tech/PropelBundle` instead would fix it. Composer looks like git@ URLs are meant to be for private repos.
https://getcomposer.org/doc/05-repositories.md#vcs